eta/core/geometry.py

```diff
@@ -31,7 +31,13 @@ class BoundingBox(Serializable):
     '''
 
     def __init__(self, top_left, bottom_right):
-        '''Constructs a BoundingBox from two RelativePoint instances.'''
+        '''Creates a BoundingBox instance.
+
```

Comment:
It would be good if we could write down the conventions for "top" and "bottom", and for "right" and "left", with respect to `numpy` array indices.  This doc string would be a good place.  E.g. I find it hard to remember with the `x` coordinate for a "left" point should be greater or less than the `x` coordinate for a "right" point, and similarly for the `y` coordinate for a "top" vs. "bottom" point.

Reply:
I added a discussion of this to the class docstring.

For the record, it would be far more confusing to adopt the opposite convention. We're simply following the same convention as NumPy, which loads images such that the top-left of the image is `[0, 0]` and the bottom-right is `[height, width]`